study.md

```diff
@@ -135,15 +135,17 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a breif description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP or  hyptertext transfer protocol is what the browswer uses to recieve
+a server address and translate the message from that sever into text in the browser
+window.
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client is and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+A client is the thing accessing the URL and the server is what recieves the message. A client types in a URL which makes a request to the server. The server processes the request and returns the URL into the broswer using HTTP.
 ```
```

Comment:
A client is the front-end user interface that is used to send requests to a
 server. Examples of clients are browsers such as Internet Explorer and Google
 Chrome or applications that act as clients.  While clients send requests and receive responses, servers receive
 these requests and send responses. A server can refer to either a physical server (computer) or a computer program hosting a server, such as the one being used in the library-api-guide.

does that clear it up at all?